docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`184` If Rotkehlchen brand name in top left clicked, open browser to rotkehlchen.io
```

Comment:
Hey @kascheri12, one small fix here: `left is clicked`. And also add at the end: `instead of showing the sign-in popup`